src/components/DocumentView/BlockSyncedBlock.tsx

```diff
@@ -13,22 +14,34 @@ export async function BlockSyncedBlock(props: BlockProps<DocumentBlockSyncedBloc
         return null;
     }
 
-    const result = await getSyncedBlock(
+    const syncedBlock = await getSyncedBlock(
         apiToken,
         block.data.ref.organization,
         block.data.ref.syncedBlock,
     );
 
-    if (!result) {
+    if (!syncedBlock) {
         return null;
     }
 
     return (
         <Blocks
-            nodes={result.document.nodes}
-            document={result.document}
+            nodes={syncedBlock.document.nodes}
+            document={syncedBlock.document}
             ancestorBlocks={[...ancestorBlocks, block]}
-            context={context}
+            context={{
+                ...context,
+                resolveContentRef: async (ref, options) => {
```

Comment:
i'm actually curious how other content refs are going to be resolved. Like if you do an `#anchor` ref to a heading in the page, it'll probably come out as `SpaceName#anchor` because the Synced Block and the Space are different.

Reply:
We will resolve in the space, but anyway we didn't decide yet about what we are doing about relative refs. See discussion with @SamyPesse in other thread.